### DISCUSSION TITLE
Custom tools

### DISCUSSION THREAD

**Discussion by IanDaly** [geoman-io/leaflet-geoman#1051] [Questions & Help] [answered]
Hello, can we implement custom tools, lets say a new shape like a Triangle ? is that possible?

Thanks

**Comment by Falke-Design** [comment, marked as answer]
Yes is possible. I made already a Triangle plugin: https://github.com/Falke-Design/GeomanTriangle

**Reply by IanDaly** [reply]
Thank you @Falke-Design

**Comment by cubecleveland** [comment]
Begiiner question ? Is there anyway to npm the package and use it in React ?

**Reply by Falke-Design** [reply]
Is it not possible to use vanilla js in react?
